shared/js/utils.js

```diff
@@ -172,6 +172,49 @@ if (inChrome) {
     callback(!!localStorage.getItem(prefName));
   }
 
+  /**
+   * Used for getting a String preference. It will either use the browser
+   * preferences or try to get them from localStorage.
+   *
+   * @param {String}   prefName The name of the preference. Note that mozLoop
+   *                            adds 'loop.' to the start of the string.
+   * @param {Function} callback Invoked with the preference value.  null is
+   *                            returned if no such preference is found.
+   */
+  function getStringPreference(prefName, callback) {
```

Comment:
Quite right; I hadn't seen that.  I was following the getBoolPreference pattern, but now I sort of assume that the point of getBoolPreference is less about the encapsulation of preferences, and more about the boolean coercion.  Weird.